src/comm/mod.rs

```diff
@@ -2,7 +2,7 @@ use bindings::*;
 use error::{Error, MPI_SUCCESS};
 
 pub fn new() -> MPI_Comm {
-  unsafe { &mut ompi_communicator_t }
```

Comment:
removed at the recommendation of rustc.
unit structs > unit enums for bindings.